src/pymatgen/io/vasp/inputs.py

```diff
@@ -1239,7 +1239,7 @@ def style(self, style) -> None:
     def automatic(cls, subdivisions: int) -> Self:
         """
         Constructor for a fully automatic Kpoint grid, with
-        gamma centered Monkhorst-Pack grids and the number of subdivisions
```

Comment:
It should be Gamma-centered according to [the VASP Wiki](https://www.vasp.at/wiki/index.php/KPOINTS)?
> The following KPOINTS file generates a regular Γ \Gamma -centered k-point.